oidc-interceptor.js

```diff
@@ -77,7 +77,7 @@ function createOidcInterceptor (options) {
       if (opts.oauthRetry) {
         return callRefreshToken()
           .catch(err => {
-            handler.onError(err)
+            handler.onResponseError(handler, err)
```

Comment:
I am not 100% sure, if the change here is correct. The tests are passing but maybe the code is not correct.

Reply:
it is